src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -243,6 +251,19 @@ const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, o
           </div>
         </Fieldset>
 
+        <Fieldset>
+          <FilterGroup>
+            <Label htmlFor="filter_notes" isGroupHeader>Zoek in notitie</Label>
+            <Input
```

Comment:
Test missing for the verification of the input element

Reply:
Sorry, should have run coverage, working on it!